opentelemetry-stdout/src/trace/transform.rs

```diff
@@ -9,17 +9,27 @@ pub struct SpanData {
     resource_spans: Vec<ResourceSpans>,
 }
 
-impl From<Vec<opentelemetry_sdk::export::trace::SpanData>> for SpanData {
-    fn from(sdk_spans: Vec<opentelemetry_sdk::export::trace::SpanData>) -> Self {
+impl
+    From<(
+        Vec<opentelemetry_sdk::export::trace::SpanData>,
+        &opentelemetry_sdk::Resource,
+    )> for SpanData
+{
+    fn from(
+        (sdk_spans, sdk_resource): (
+            Vec<opentelemetry_sdk::export::trace::SpanData>,
+            &opentelemetry_sdk::Resource,
+        ),
+    ) -> Self {
```

Comment:
might as well create a new constructor instead of a tuple from here

Reply:
yes `From` with tuple is somewhat weird. Created the new instead. Thanks.